src/main/resources/META-INF/openapi.yaml

```diff
@@ -159,7 +154,8 @@ components:
             $ref: '#/components/schemas/TransitiveDependencyReport'
         recommendation:
           description: Trusted Content recommendation that is not related to any security vulnerability
-          $ref: '#/components/schemas/PackageRef'
+          allOf:
+            - $ref: '#/components/schemas/PackageRef'
```

Comment:
Thanks, this was missing because attributes were added to the property definition

Reply:
Yeah I think the second one, the _highestVulnerability_ was my bad. :-)